include/mbgl/map/camera.hpp

```diff
@@ -139,21 +139,21 @@ struct FreeCameraOptions {
     std::optional<vec4> orientation = std::nullopt;
 
     /** Helper function for setting the mercator position as Lat&Lng and altitude in meters */
-    void setLocation(const LatLngAltitude& location);
+    void setLocation(const LatLngAltitude& location) noexcept;
```

Comment:
`mbgl::util::clamp` (called somewhere down) is not `noexcept`. But it can be made `noexcept`.